src/Share/Survey/SurveyHelper.cs

```diff
@@ -78,213 +97,62 @@ public static SurveyHelper GetInstance()
             }
             return _instance;
         }
-
-        public bool ShouldPropmtSurvey(string moduleName, Version moduleVersion)
-        {
+        public bool ShouldPropmtAzSurvey(String installationId){
```

Comment:
Do you need `installationId` to determine survey should be prompt or not? If no, this parameter should be removed.